src/_posts/languages/ruby/rails/2000-01-01-start.md

```diff
@@ -149,7 +149,7 @@ but it's expected. There is nothing in the project, it's time to build your prod
 
 Latest version of Rails application also use Node.js in case ExecJS or
 Webpacker is detected. The Node.js version built inside the Ruby buildpack is
-frozen. It is the latest version of the LTS 16.x at the time of writing.
+frozen. It is the latest version of the LTS 20.x at the time of writing.
```

Comment:
issue: this is not correct. The Ruby buildpack embeds a simple support of Node.js and in this context v16.x is installed by default:

https://github.com/Scalingo/ruby-buildpack/blob/2aba1378b09270d817189dd1368fbd66344db36f/lib/language_pack/helpers/nodebin.rb#L5

Reply:
Right, I was anticipating the `ruby-buildpack` update :sweat_smile:
Should be reverted now :)